app/css/base.css

```diff
@@ -1,6 +1,6 @@
 @font-face {
   font-family: "gaia-icons";
-  src: url("../media/fonts/gaia-icons.ttf") format("truetype");
+  src: url("../media/fonts/gaia-icons.woff") format("truetype");
```

Comment:
I assume the woff only work in FxOS? In Fx34 I now see the word sync spinning ;) and also the other icons are missing and replaced by text. Perhaps we can keep both the ttf and woff?